modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -60,424 +61,56 @@ with links to every release package along with precompiled binaries.
 
 [source,bash]
 ----
-wget https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-linux-2.7.0.gem
-gem install couchbase-3.2.0-x86_64-linux-2.7.0.gem
+wget https://packages.couchbase.com/clients/ruby/sdk-3.3.0/couchbase-3.3.0-x86_64-linux-2.7.0.gem
+gem install couchbase-3.3.0-x86_64-linux-2.7.0.gem
 ----
 
 
 [#latest-release]
-== Version 3.2.0 (4 August 2021)
```

Comment:
@avsej thanks for this! 
Quick one, not sure we should be removing/overriding older releases...
I'll put them back in if that's ok?

Reply:
yes, sure. feel free to put it back.